server/routes/settings.js

```diff
@@ -0,0 +1,134 @@
+const express = require("express");
+const router = express.Router();
+const mongoose = require("mongoose");
+const SettingsModel = require("./../models/Settings");
+const UserModel = require("./../models/User");
+const { jwtVerify } = require("../utils/authUtils");
+
+mongoose.connect(process.env.MONGO_URL, { useNewUrlParser: true  });
+
+const saveSettings = (settings, res) => {
+  settings.save((err) => {
+    if (err) {
+      next(err);
+    }
+  });
+  res.send({ error: false, settings });
```

Comment:
I think the default I set up would be to send { success: (true/false), ...} as opposed to { error: (true/false), ...}. Can we switch this to avoid ambiguity? So res.send({ success: true, settings });